daisy/step.go

```diff
@@ -48,6 +48,7 @@ type Step struct {
 	CreateInstances        *CreateInstances        `json:",omitempty"`
 	CreateNetworks         *CreateNetworks         `json:",omitempty"`
 	CopyGCSObjects         *CopyGCSObjects         `json:",omitempty"`
+	StopInstances          *StopInstances          `json:",omitempty"`
```

Comment:
Would you also need startinstances? If you don't need it for you tests there's no need to add it now, though we proabbly should make an issue to track it.

Reply:
I don't need startinstances for my tests, I'll make an issue then